### PR TITLE
Adding the revised html files

### PR DESCRIPTION
**Pull request by cwilliams366**
I looked it over and seems nothing should overwrite anything, I know I deleted the html files that weren't necessary and redid the ones that to be more simpple. 

I know this branch doesn't have the uploaded_files folder so I hope it doesn't do anything to the main branch but yea review and see how it looks

### CODE REVIEW

**Comment by cwilliams366** [comment]
I thought I deleted the self_service_resource page....that's something we don't need. I changed it to self_help_services

File: templates/self_service_resources.html [hunk 1]
```diff
@@ -37,30 +35,32 @@
...
             <div class = "multicolumns">
               <ul style = "border-style:outset;border-width:8px;margin:-1px;height:250px">
                 <b style = "border-style:solid;border-width:3px;margin:100px;">Academic Assistance</b>
                 <br>
                 
                 <li> <a href="tutors_portal_page.html"><b>Tutors Portal</b></a></li>
-                <li> <a href="course_catalog_page.html"><b>Course Catalog</b></a></li>
-                <li> <a href="academic_clubs_workshops_page.html"><b>Academic Clubs/Workshops</b></a></li>
+                <li> <a href="https://catalogue.howard.edu/classes"><b> Academic Catalog</b></a></li>
+                <li> <a href="https://howard.campuslabs.com/engage/organizations"><b>Academic Clubs/Workshops</b></a></li>
```
Comment (anniamatthews) [inline templates/self_service_resources.html:50]: 👍🏽

File: templates/university_lab_directory.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html>
```
Comment (anniamatthews) [inline templates/university_lab_directory.html:1]: This code is nicely indented and formatted so we don't have to worry about readability👍🏽

**Review by anniamatthews — APPROVED** [review]

**Comment by cwilliams366** [comment]
Bet. Thanks making sure, we cookin' is all

**Pull request merged**